### DISCUSSION TITLE
Css Theme : How to customize?

### DISCUSSION THREAD

**Discussion by dominiclapointe** [oqtane/oqtane.framework#845] [Q&A] [answered]
Hi all,

Quick question: If i am to start a new project using Oqtane, how can i set a corporate business template or lets say a html5 admin template from theme forest (as a lot of dev do i guess).

The different module can have their own html / css but what about the core? Is there a way to customize it? Top navigation bar with companie logo and specific color, left hand side menu etc...

That might sound stupid as a question but i cant figure out how that can be achieve. Even though the black 'template' is nice, that doesnt work for any corportate project or even a project with a visual identity that must be respected.

Thank you

**Comment by mikecasas** [comment, marked as answer]
One quick solution is to go into your site settings and change the theme to the Blazor Theme.  This gives you the standard Blazor layout with the left side-bar and a top navigation bar.

Oqtane is based on Bootstrap so you can use your own custom Bootstrap css to change the colors.

In the end you may want/need to create your own theme to match your specific branding.

**Comment by sbwalker** [comment]
@dominiclapointe you can look at an example custom theme project here:

https://github.com/oqtane/oqtane.theme.cerulean

It uses the same HTML structure as the default Oqtane template but uses the Bootswatch Cerulean styles. You can modify the HTML structure by modifying the theme components ie. Default.razor

**Comment by dominiclapointe** [comment]
Thank you, all make sense now!
